src/backend/api/helpers.ts

```diff
@@ -9,6 +9,7 @@ export const openSupportPage = () => ipcRenderer.send('openSupportPage')
 export const quit = () => ipcRenderer.send('quit')
 export const showAboutWindow = () => ipcRenderer.send('showAboutWindow')
 export const openDiscordLink = () => ipcRenderer.send('openDiscordLink')
+export const openWinePrefixFAQ = () => ipcRenderer.send('openWinePrefixFAQ')
```

Comment:
This got lost at some point https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/pull/1774